crates/compiler/types/src/subs.rs

```diff
@@ -3207,64 +3225,55 @@ fn occurs(
     let root_var = subs.get_root_key_without_compacting(input_var);
 
     if seen.contains(&root_var) {
-        Err((root_var, vec![]))
+        Err((root_var, Vec::with_capacity(0)))
```

Comment:
are these different in practice? I agree that `with_capacity(0)` expresses the intent more clearly here, but I'd expect that `vec![]` would effectively do the same thing?

Reply:
oops, this was a leftover from something else I tried.